.aspect/bazelrc/BUILD.bazel

```diff
@@ -0,0 +1,5 @@
+"Aspect bazelrc presets; see https://docs.aspect.build/guides/bazelrc"
```

Comment:
Is any of this necessary for bzlmod? Can we do it before this PR?

Reply:
https://github.com/aspect-build/rules_cypress/pull/62